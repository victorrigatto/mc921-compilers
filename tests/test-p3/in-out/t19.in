void main() {
  int i = 0;
  if(i)
    return 1;
  else
    return 0;
}
