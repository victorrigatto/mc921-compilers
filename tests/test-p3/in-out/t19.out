SemanticError: The condition expression must be of type(bool) @ 3:6
